src/Taxjar/TaxjarApi.cs

```diff
@@ -95,20 +95,20 @@ protected virtual RestRequest CreateRequest(string action, Method method = Metho
             {
                 if (IsAnonymousType(body.GetType()))
                 {
-                    request.AddJsonBody(body);
-
                     if (method == Method.GET)
                     {
                         foreach (var prop in body.GetType().GetProperties(BindingFlags.Instance | BindingFlags.Public))
                         {
                             request.AddQueryParameter(prop.Name, prop.GetValue(body).ToString());
                         }
                     }
+                    else
```

Comment:
I would consider checking if it's one of the allowed instead - `PUT`, `PATCH`, or `POST`.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

Reply:
Agreed, since it's conventional to use query params with DELETE. Updated!